x-pack/test/fleet_api_integration/apis/package_policy/create.ts

```diff
@@ -47,230 +46,229 @@ export default function ({ getService }: FtrProviderContext) {
         .send({ agentPolicyId });
     });
 
-    it('should fail for managed agent policies', async function () {
-      if (server.enabled) {
```

Comment:
Most of the changes here are whitespace only from replacing the `if (server.enabled) { ... } else { ... }` pattern with `skipIfNoDockerRegistry`.